code/UserForm.php

```diff
@@ -210,7 +219,12 @@ public function getCMSFields()
                 CheckboxField::create(
                     'DisableSaveSubmissions',
                     _t('SilverStripe\\UserForms\\Model\\UserDefinedForm.SAVESUBMISSIONS', 'Disable Saving Submissions to Server')
-                )
+                ),
+                TreeDropdownField::create(
+                    'ConfirmationPageID',
+                    _t('SilverStripe\\UserForms\\Model\\UserDefinedForm.CONFIRMATIONPAGE', 'Custom confirmation page'),
+                    SiteTree::class
+                ),
```

Comment:
This formfield should already be scaffolded as per `SiteTree::scaffoldFormFieldForHasOne()`. Is there a need to redeclare it here?